Zend/tests/attributes/019_variable_attribute_name.phpt

```diff
@@ -8,4 +8,4 @@ class A {}
 
 ?>
 --EXPECTF--
-Parse error: syntax error, unexpected '$x' (T_VARIABLE), expecting identifier (T_STRING) or static (T_STATIC) or namespace (T_NAMESPACE) or \\ (T_NS_SEPARATOR) in %s on line %d
+Parse error: syntax error, unexpected variable "$x", expecting identifier or "static" or "namespace" or "\" in %s on line %d
```

Comment:
Could the `syntax error` start with a capital letter? If it's not too difficult to achieve then I'd love it, since a lot of messages have been capitalized for PHP 8. If this change would require a non-negligible amount of work, please ignore this request :)

Reply:
@kocsismate I looked at that, and unfortunately that string comes from a different function somewhere deep in Bison's generated code. :( 

The best approach I can think of is to copy it to a new string and fix the capital letter somewhere in the process of creating the Error object.

Reply:
If it requires quirks like that then I think we can also live with a lowercase letter :)